core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/HttpChecksumStage.java

```diff
@@ -74,6 +75,11 @@ public SdkHttpFullRequest.Builder execute(SdkHttpFullRequest.Builder request, Re
             return request;
         }
 
+        // If SRA is enabled, we skip flexible checksum in header
```

Comment:
Can we add a TODO to indicate that we should rewrite this codepath to remove flexible checksums in headers once the released code is stable?